pkgs/applications/version-management/subversion/default.nix

```diff
@@ -49,6 +49,14 @@ let
     # "-P" CPPFLAG is needed to build Python bindings and subversionClient
     CPPFLAGS = [ "-P" ];
 
+    env = lib.optionalAttrs stdenv.cc.isClang {
+      NIX_CFLAGS_COMPILE = toString [
```

Comment:
I forget why but IIRC `toString` on lists is cursed; best to explicitly `concatStringsSep " "`.